### PR TITLE
fix: internal packages now takes suffix and esm opt.

### PR DESCRIPTION
**Pull request by siendoricardo**
This PR fixes: #615

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -198,6 +198,13 @@ export function getPropertyAccessor(objectName: string, propertyName: string, op
     : `${objectName}${optional ? '?.' : ''}[${JSON.stringify(propertyName)}]`;
 }
 
+export function impSlash(options: Options, spec: string) {
+  if (!spec.includes('/'))
```
Comment (paralin) [inline src/utils.ts:202]: There are a lot of imports like `@foo/bar` which have a slash but are not referencing a specific file.

I don't see why it's necessary to add another function `impSlash` as opposed to just appending `importSuffix` everywhere `protobufjs/minimal` is imported?

**Review by paralin — CHANGES_REQUESTED** [review]

File: src/utils.ts (hunk 1 above)
Comment (siendoricardo) [reply src/utils.ts:202]: That's what the function it's doing. I just thought it looks cleaner to have it as util. What might be redundant are the includes check.

Comment (paralin) [reply src/utils.ts:202]: Maybe call it `impFile` instead of `impSlash` and remove the `/` check ?

Comment (siendoricardo) [reply src/utils.ts:202]: That sounds great. 👍

**Comment by siendoricardo** [comment]
I'll fix the lint.

**Review by paralin — APPROVED** [review]

**Comment by paralin** [comment]
@siendoricardo Squashed your commits to 1 commit & added some more info in the commit description, hope that's ok w/ you!

@stephenh This looks good to me.

**Review by stephenh — APPROVED** [review]
Looks great, thanks @paralin and @siendoricardo !

**Pull request merged**